gqueries/output_elements/output_series/table_260_land_use_solar_wind/solar_pv_buildings/land_use_solar_wind_solar_pv_buildings_total_land_use.gql

```diff
@@ -0,0 +1,9 @@
+- query = 
+    PRODUCT(
+        DIVIDE(
+            Q(land_use_solar_wind_solar_pv_buildings_land_use_per_unit),
+            MILLIONS
+        ),
+        Q(land_use_solar_wind_solar_pv_buildings_number_of_units)
+    )
+- unit = km2
```

Comment:
Why are you first converting `land_use_solar_wind_solar_pv_buildings_land_use_per_unit` to m2 in the query, and then reconverting it to km2 here?